### DISCUSSION TITLE
Why on my website I see my README.md

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#34804] [Other Feature Feedback, Questions, & Ideas] [answered]
Why on my website I can't see my pictures?

website: https://mitkovtori.github.io/Mitko.Vtori-personal-blog/

repo: https://github.com/MitkoVtori/Mitko.Vtori-personal-blog

**Comment by jfullstackdev** [comment, marked as answer]
your `index.html` is not the same as your README, 

as you can see, you don't have all of those images present in your README in your html

![image](https://user-images.githubusercontent.com/47092464/193440397-15aa398a-9a14-459d-9da5-d22948ab0b81.png)

but you can successfully load the images that you want by embedding it too, you need to update your html code for that

again, the one being loaded by `.github.io` is your `index.html` and the one on the repo is the README,
